package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "my-component",
-  "componentName": "MyComponent",
+  "name": "@decsys/horizontal-component",
+  "componentName": "Horizontal",
```

Comment:
We're gonna want a better name ;)

spaces are acceptable.

I think I suggested something like "Multiple Choice" earlier? But open to suggestions.

"Select" or "One Of" or something might also work. But Horizontal has nothing too do with the kind of response they give, or the means of giving.

Purely a description of its layout which is probably also wrong sometimes (e.g. for a dropdown box, or a vertical list o radios?)